server/sonar-server/src/test/java/org/sonar/server/issue/index/IssueIndexTest.java

```diff
@@ -519,43 +520,47 @@ public void filter_by_resolved() {
   }
 
   @Test
+  @Ignore
   public void filter_by_rules() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Fix or remove this skipped unit test ([squid:S1607](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Feh%2FSONAR-10303&issues=AWEx4Fzrr8h_fVHZ4pDU&open=AWEx4Fzrr8h_fVHZ4pDU)

[ISSUE_KEY]: AWEx4Fzrr8h_fVHZ4pDU